mne/io/bti/bti.py

```diff
@@ -893,7 +893,8 @@ def _read_bti_header_pdf(pdf_fname):
 def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
     """ Read bti PDF header
     """
-    info = _read_bti_header_pdf(pdf_fname) if pdf_fname else dict()
+    info = _read_bti_header_pdf(pdf_fname) if pdf_fname else \
+        dict({'sample_period': 0.001})
```

Comment:
Had a short chat with @dengemann . The reason for having None as sfreq is because sometimes it is useful to only read the header info and for bti files you have to read the data to find out the sfreq.

Would it be okay to change it so that in ``_empty_info`` would also accept None as sfreq? Could ``lowpass`` also be None in that case?